include/uapi/sound/sof/tokens.h

```diff
@@ -133,6 +132,7 @@
 
 /* Processing Components */
 #define SOF_TKN_PROCESS_TYPE                    900
+#define SOF_TKN_PROCESS_PAYLOAD_WITH_OUTPUT_FMT 901
```

Comment:
Let's hope that we will not forget to update the commit subject and message when squashing the fixup for upstream...